composer.json

```
@@ -67,6 +67,7 @@
         "rector/rector": "^0.19",
         "symfony/console": "^5.4 || ^6.0 || ^7.0",
         "symfony/phpunit-bridge": "^6.0 || ^7.0",
+        "symfony/uid": "^7.0",
```

```suggestion
        "symfony/uid": "^5.4 || ^6.0 || ^7.0",
```

Need at least 5.4 for PHP 7.4 compat